server.js

```diff
@@ -17,8 +17,12 @@ const configPath = process.env.SERVER_EXTRA_CONFIG_FILE || '/etc/webapp/customSe
 const DEFAULT_CUSTOM_CONFIG_DATA = {
   services: []
 };
+
+const walkthroughLocations = process.env.WALKTHROUGH_LOCATIONS || './public/walkthroughs';
+
 const CONTEXT_PREAMBLE = 'preamble';
 const CONTEXT_PARAGRAPH = 'paragraph';
+const LOCATION_SEPARATOR = ';';
```

Comment:
Was there a reason a comma couldn't be used here? (Only asking because I'd be used to comma separated lists)

Reply:
A comma would be fine too. I was thinking of bash where a semicolon is used.